### ISSUE TITLE
Calling for Maintainers

### ISSUE THREAD

**Issue by domaindrivendev** [domaindrivendev/Swashbuckle#1053]
With the introduction of [ASP.NET Core](https://www.asp.net/core), I've now shifted my focus to the Core-specific project - [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore). That project will be receiving most of my (already limited) personal time, and so I won't have the capacity to maintain this one at a sufficient rate. Still, I'd love to see it live on and am seeking one or two "core" contributors / maintainers to help out. Ideally, these would be people who have already contributed through PRs and understand the inner workings and overall design. Once signed-up, we can agree on an approach that works - ultimately, I want to remove myself as the bottleneck to merging PRs and getting fresh Nugets published. If you're interested, please let me know by adding a comment below:

**Comment by heldersepu** [comment]
Hello Hello, I just noticed this call..
Yes this project has a lot of good Pull request that can be merged as is or after solving some small conflicts.
What can I do to help?

**Comment by svrooij** [comment]
Sign me up.

I've been using this package for a long time, and think I could be a good help in maintaining all the pull-requests.

**Comment by lockewritesdocs** [comment]
I am interested in helping to support Swashbuckle documentation (I'm a tech writer). I implemented Swashbuckle for one of our C# APIs and think it's rather fantastic, so let me know where I can assist!

**Comment by heldersepu** [comment]
@lockewritesdocs
I think that the long [readme](https://github.com/domaindrivendev/Swashbuckle/blob/master/README.md) could be split into multiple wiki pages...
What do you think?

**Comment by domaindrivendev** [comment]
Awesome - I'm thrilled to see these responses. @heldersepu - I'm especially grateful for all your issue responses and PR's over the last few months.

For next steps, how do people feel about getting on a Google hangout and talking in person? There's some planning to be done!

**Comment by heldersepu** [comment]
@domaindrivendev
Sure; I'm fine with talking/typing.
 when do we do it?

**Comment by heldersepu** [comment]
@domaindrivendev 
I think that the first order of business will be merge those pending Pull requests
My favourite is #756 that has been a thorn in my side for a while

**Comment by domaindrivendev** [comment]
Yeh - I'm fairly opinionated about what should be included with the core package and what should be left to application developers via the wide range of extensibility options.

So ... I want to triage and apply that lense. Of course on the flip side, I haven't had the bandwidth to provide feedback and I'm sure that's frustrating for contributors. Hence my desire to get us on the same page before opening the PR floodgates

**Comment by heldersepu** [comment]
@domaindrivendev
I see your point, you have put a lot of time and effort into this project and want to make sure it stays on the right course.

Frustration from contributors... there is no doubt but I don't think is that  much, everyone knows this is an OpenSource project without any major financial backers, we all have full time jobs somewhere.
Anyone looking for a quick turnaround on a new feature or bug fix, can fork the repo and run their own version (if they can code & have time), if not they can hire a developer to do it for them.

**Comment by lockewritesdocs** [comment]
@heldersepu, I agree that the README can be broken out into separate pages. Additionally, I think that a short "Getting started" tutorial with more in-depth examples could be incredibly beneficial.

**Comment by heldersepu** [comment]
@lockewritesdocs
I'm 100% with you! a short "Getting started" tutorial is very helpful and example are awesome, I've been collecting a few here:
https://github.com/heldersepu/csharp-proj/tree/master/Swagger_Test

**Comment by lockewritesdocs** [comment]
@heldersepu, sounds good. However, the link that you provided is broken, and I could not find the Swagger_Test project in https://github.com/heldersepu/csharp-proj.

Also, I added code to modify Swashbuckle so that it generates a swagger.json file, which I then served up to [DapperDox](http://dapperdox.io/). It's a pretty cool integration, and I wonder if it's worth documenting. Thoughts?

**Comment by heldersepu** [comment]
@lockewritesdocs 
Yes it is broken, sorry about that, I decided to move it to own project:
https://github.com/heldersepu/SwashbuckleTest

For your modification, you should open a new issue document there what you are trying to accomplish and send a Pull request.

**Comment by domaindrivendev** [comment]
@lockewritesdocs - regarding use of the wiki, I'm all for splitting up the docs for improved readability but I'm on the fence about wiki vs readme (or collection of) that are source controlled along with the code. I prefer the latter, because I like PR's (especially feature enhancements) to encapsulate a "complete package", including any required doc updates. Unless I'm mistaken, source control and the ability to incorporate into PR's isn't possible with a WIKI? On the flip side, I could be convinced if you have other compelling reasons :)

@heldersepu - thanks again for all you're continued work. I know I need to give you more support and at a minimum give you the ability to merge and release to Nuget. Like I said, I just wanted to align a little before doing so. What's next week like for you - Would you be free to chat for an hour Mon or Tues around 4:30 PST. @lockewritesdocs & @svrooij you're also both welcome to join too.

Regarding contact details, it would make most sense for you guys to connect with me over linkedin (https://www.linkedin.com/in/richard-morris-08479821/), then we can share emails for a Google hangout.

**Comment by heldersepu** [comment]
@domaindrivendev 
I'm not sure about 4:30PM PST... I'm in FL that's 7:30PM, after 7pm is baby time (feeding, shower & putting her to sleep) sometimes she helps an everything is quick & painless, some others every step is a fight.
Will try let's see what happens

**Comment by heldersepu** [comment]
@domaindrivendev 
I think we should start encouraging users to move all the Q/A to stackoverflow ...
The issues are flooded with questions I don't think that is the place for that.

**Comment by lockewritesdocs** [comment]
I've been absent for quite some time; lots of things happening in my world right now.

Have either of you thought about using [GitHub pages](https://pages.github.com/) for the documentation? That implementation enables publishing docs from the source code, circumventing the problem that @domaindrivendev mentioned with a wiki. I think that the README has a lot of valuable information, but needs to be broken up into separate sections. Having that much content in a README is overwhelming IMO. A README is meant to provide a brief description of the tool, what it does, and how to get started, whereas the remaining information could be well organized in GitHub pages.

Thoughts?

**Comment by sachinjegaonkar** [comment]
I would like to contribute to this by any of the means that you see. I have used Swashbuckle in one of our projects. Enthusiastic to get a chance to work on this kind of project. Primarily, I use C++ and C# for my work. Let me know how can I help you.

**Comment by monemihir** [comment]
Would love to help out. Been using Swagger for a long time with Swashbuckle.

Would also be very interested in helping out with the AspNetCore package since we are moving to AspNetCore as well for the product which uses Swashbuckle. Currently we have it running happily through .Net Mono but have to maintain 2 code bases (not because of Swashbuckle) due to some idiosyncrasies of Mono.

**Comment by PaulDotNet** [comment]
I want to get my hands on AspNetCore. There are some basic things that have to be merged but they are stuck,  Sign me up.

**Comment by mkondaveeti** [comment]
Richard , I  have good experience in  using swashbuckle. I had experience in overriding  message hanndler and implementing filters for one of my clients for customtoken url and taking [] from generic types.
I have many times debugged entire swash buckle code for customization. Please sign me up.

**Comment by FrameBassman** [comment]
I want to help

**Comment by Alfetta159** [comment]
Let me know if I can help. I'm starting to use Swashbuckle for Core as I try to use .NET Core as much as possible.

**Comment by lonesomegeek** [comment]
I'm interested, needing help?

**Comment by gbaychev** [comment]
Hi everybody,
I just came by to ask what the current status is? Are there enough maintainers, is the project dead, what is happening? It seems that the last commit on master was a year ago.

Paging Mr. @domaindrivendev 

Regards

P.S I don't mean any disrespect, I'm just curious what the situation is

**Comment by gbaychev** [comment]
Ok, dead it is then.

**Comment by domaindrivendev** [comment]
I'm devoting 100% of my _spare_ dev cycles to the [ASP.NET Core version of this tool](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) and therefore am unable to continue maintaining this particular repo. So, I see two potential options:

1.  Completely handover continued maintenance to one or more of the volunteers from this thread (@svrooij, @lockewritesdocs, @sachinjegaonkar @monemihir @mkondaveeti, @FrameBassman, @Alfetta159, @lonesomegeek). Please let me know if anyone from that group has the capacity to drive such an effort.

OR

2. Sunset this repo and redirect interest to an active fork such as [Swagger-Net](https://github.com/heldersepu/Swagger-Net).

**Comment by lockewritesdocs** [comment]
I changed jobs about 1.5 years ago and have not used Swashbuckle since leaving the previous company. I definitely cannot maintain this project.

**Comment by jv42** [comment]
@domaindrivendev I think the second option would be best, as there is indeed an active fork. 
Of course, that's only my opinion ;-)

**Comment by OskarKlintrot** [comment]
@heldersepu has already done a lot of work on [Swagger-Net](https://github.com/heldersepu/Swagger-Net) so that would indeed make sense. IMHO :)

**Comment by gbaychev** [comment]
Alright guys, thank you for the heads-up.

Also big thanks for @domaindrivendev . Just wanted to let you know that the swagger is one of the most beloved features in our REST APIs, especially among our QA and 3rd party and this is thanks to you.

**Comment by robinrodricks** [comment]
I am willing to maintain this project. I already took over FluentFTP from the original author three years ago and its now blossoming just fine, with tons of new features, PRs, issues fixed, nuget package, etc. @domaindrivendev Please contact me if you are willing to handover.

**Comment by Calebyang93** [comment]
Hi @domaindrivendev I will be interested to help to maintain the codebase if there are issues. Will read up on documentation and practice if you need help.

**Comment by TheMrAnderson** [comment]
@domaindrivendev I use this in my current job as we have some APIs that are too troublesome to move to .Net Core.  I would be interested in helping to maintain this and even upgrade to Swagger 3.0.
